.github/workflows/themes-weekly-pull.yml

```
@@ -0,0 +1,46 @@
+name: themes-weekly-pull
+
+on:
+  schedule:
+    # run every friday at 5AM
+    - cron: '0 5 * * 5'
+
+jobs:
+  pull-from-jsonbin:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Node.js 14.x
+        uses: actions/setup-node@v2

```

```suggestion
        uses: actions/setup-node@v3
```
Unless there is a specific reason for using the outdated action version, better to use the latest major version ref:

https://github.com/actions/checkout/tags